src/website/app/demos/Button/examples/index.js

```diff
@@ -4,6 +4,7 @@ import regular from './regular';
 import circular from './circular';
 import icons from './icons';
 import iconOnly from './iconOnly';
+import iconsWithTruncation from './iconsWithTruncation';
```

Comment:
Rather than add a new "dev-only" example for "buttons with icons with truncation", you could simply add the additional buttons with icons to the existing truncation example.  Users might be curious to see how truncation behaves in these scenarios as well.

Reply:
Good idea, I'm on it.